ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/HelpIconTh.tsx

```diff
@@ -1,23 +1,28 @@
-import React from 'react';
-import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import React, { ReactElement } from 'react';
+import { Flex, FlexItem, Popover } from '@patternfly/react-core';
 import { Th, ThProps } from '@patternfly/react-table';
 import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
 
 type HelpIconThProps = {
     children: string | React.ReactNode;
     sort?: ThProps['sort'];
-    tooltip: string;
+    popoverTitle: string;
+    popoverContent: ReactElement;
 };
 
-function HelpIconTh({ children, tooltip, sort }: HelpIconThProps) {
+function HelpIconTh({ children, sort, popoverTitle, popoverContent }: HelpIconThProps) {
     return (
         <Th sort={sort || undefined}>
             <Flex direction={{ default: 'row' }} alignItems={{ default: 'alignItemsCenter' }}>
                 <FlexItem>{children}</FlexItem>
                 <FlexItem>
-                    <Tooltip content={tooltip}>
+                    <Popover
+                        aria-label="Table column help text"
```

Comment:
Should the `aria-label` prop go on the icon instead, so that it is available before a user clicks on the icon?

Reply:
We should look at the rendered markup, because by putting it on the PF component, it may be transferred to the trigger button, which may be even better than putting it directly on the icon?

Can run Axe browser extension to check.

Suggestion: Be specific with the column name in the aria-label, which would require another prop, I think

Reply:
Ok, so turns out that adding a `title` to the `Popover` cancels out the `aria-label`. So I did the following:
1. Removed the `title` and kept the `aria-label` the same as above
2. I added an `aria-labelledby` for the icon that says `Table column help icon` to differentiate it
Does that work for you?

Reply:
Does the markup look sane, and does it pass Axe?

Reply:
If you look at the props for `Popover` there are only two props that deal with aria-labels: `closeBtnAriaLabel` and `aria-label`. Neither deals with the trigger element, unfortunately. The `children` for `Popover` ends up being the trigger. I've done at least another adjustment to wrap the icon in a button and but the aria-label on that to be more correct.

If you'd like to see the mark up, this is what it shows

![Screenshot 2023-08-08 at 2 49 42 PM](https://github.com/stackrox/stackrox/assets/4805485/49ebf6f6-4a69-481d-8278-456322232dd9)
![Screenshot 2023-08-08 at 2 50 10 PM](https://github.com/stackrox/stackrox/assets/4805485/b7a2b020-a93d-4560-a981-ee64b7189033)

Reply:
Thanks for looking into that, @sachaudh